### ISSUE TITLE
Update taco packs automatically

### ISSUE THREAD

**Issue by bluenote73** [gw2-addon-loader/GW2-Addon-Manager#164]
I'd love it if taco marker packs could be included in updating. So taco/blish marker pack auto updates for the big marker packs that are regularly updated. Like Tekkit, reactif, fast farm community, tyriangathering project. thanks for considering!

**Comment by Friendly0Fire** [comment]
I've been talking with @dlamkins on how we might handle Blish updates. He's still not sure whether he won't just have Blish handle updates on its own, but we'll see what happens!

**Comment by dlamkins** [comment]
_tl;dr it seems like it'll probably be easiest to keep this kind of functionality in the overlays themselves and we've made lots of progress towards it in Blish HUD._

We've been tracking marker pack updates and more at https://mp-repo.blishhud.com/ that we intend to utilize for our upcoming repo features.  Only a fraction of packs get released through GitHub or other standardized hosting sites, so it currently requires a 3rd party (so us, in this case) to confirm when there are changes (we review all packs for potential changes, depending on their platform, ever 30 or so minutes).

You can see how we intend to have this accessible in-game via Blish HUD in our upcoming repository functionality which will allow for auto updates.

![image](https://user-images.githubusercontent.com/1950594/142070638-e4162f66-230a-405b-86ce-4decfdbcd5fb.png)

This doesn't necessarily solve the issue for TacO, but other services are more than welcome to use our repo data (https://mp-repo.blishhud.com/repo.json though, do note, it's format, fields, etc. are not finalized at this time).

As we're made aware of additional packs, we add them easily to the repo so that they're included in the next update cycle.

**Comment by Friendly0Fire** [comment]
Sounds awesome as always. I'll close this since I think you guys have this in the bag. If we ever get to the point of having the addon manager handle external applications, we can see if Blish could be one of them, but that'll be a different issue.
